custom/enikshay/private_sector_datamigration/management/commands/create_cases_by_beneficiary.py

```diff
@@ -167,3 +225,23 @@ def handle(self, domain, **options):
         logger.info('Setting cleanliness flags')
         set_cleanliness_flags_for_domain(domain, force_full=True, raise_soft_assertions=False)
         logger.info('Done!')
+
+    @staticmethod
+    def perform_checks(beneficiaries):
```

Comment:
moved to ```CHECK``` constraints: https://github.com/dimagi/commcare-hq/pull/16342/commits/13dfd799bfeea7c9259fd9d69dcec321c5736a8f